searchlib/src/vespa/searchlib/attribute/enum_store_types.h

```diff
@@ -4,14 +4,14 @@
 
 #include <vespa/searchcommon/attribute/iattributevector.h>
 #include <vespa/vespalib/datastore/entryref.h>
-#include <vespa/vespalib/util/array.h>
+#include <vespa/vespalib/stllike/allocator.h>
 
 namespace search::enumstore {
 
 using Index = vespalib::datastore::EntryRef;
 using InternalIndex = vespalib::datastore::EntryRefT<22>;
-using IndexVector = vespalib::Array<Index>;
+using IndexVector = std::vector<Index, vespalib::allocator_large<Index>>;
```

Comment:
Should `<vector>` be added as an explicit include? Currently transitively included via `iattributevector.h`, but this is a bit opaque

Reply:
There will be follow ups cleaning up includes a bit more. Will probably happen then.